project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.module

```diff
@@ -119,6 +144,29 @@ function c4m_features_og_group_dashboard_action_promote($node) {
 }
 
 /**
+ * Callback to show the recomendation link.
+ *
+ * @param object $node
+ *    The node object.
+ *
+ * @return string
+ *    Rendered output of the promote link.
+ */
+function c4m_features_og_group_dashboard_action_recommend($node) {
+  $flag_link = flag_create_link('c4m_og_content_recommend', $node->nid);
+  $output = theme('html_tag', array(
```

Comment:
@nirgn975 
We should check the returned value from `flag_create_link` since there are 2 cases where it returns nothing ...
Something like:
```PHP
  if (!$flag_link) {
    return '';
  }
```